atef/widgets/config/utils.py

```diff
@@ -535,9 +536,13 @@ def paintEvent(self, e: QPaintEvent):
         # the handle will move along this line
         trailLength = contRect.width() - 2 * handleRadius
         xPos = contRect.x() + handleRadius + trailLength * self._handle_position
-        iconRad = 0.7 * handleRadius
+        iconRad = int(0.7 * handleRadius)
         # center of handle
-        icon_x = xPos - (1.3 * handleRadius) + iconRad
+        icon_x = int(xPos - (1.3 * handleRadius) + iconRad)
```

Comment:
```suggestion
        icon_x = int(xPos - (1.3 * handleRadius) + (1.3 * iconRad))
```

For me this helps center the icon inside the handle, do you mind checking to make sure that holds true in your env?  I don't know why it shouldn't but maybe something weird happens in qt6...

Reply:
As per my other comment...apparently I'm not using Qt6 at all! Sorry for the confusion!

Reply:
I would still suggest this change though, just to center the icon in the handle a bit more.  👍

Reply:
Sorry for the delay! I've applied this change now